lib/model/survey_model.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter_survey/api/response/survey_response.dart';
+
+class SurveyModel extends Equatable {
+  final String id;
+  final String title;
+
+  const SurveyModel({required this.id, required this.title});
+
+  @override
+  List<Object?> get props => [id, title];
+
+  factory SurveyModel.fromResponse(SurveyResponse response) {
+    return SurveyModel(id: response.id ?? "", title: response.title ?? "");
```

Comment:
1 more field so this should be re-formatted with each field in each line of code 🙏